### ISSUE TITLE
Loading pre-trained model 

### ISSUE THREAD

**Issue by msalhab96** [msalhab96/SpeeQ#46]
### Description

Currently, the optimizer parameters get loaded if a pre-trained model passed, it would be better  if we Add an option to reset the optimizer learning rate when loading it
